corgi/tasks/errata_tool.py

```diff
@@ -35,6 +35,8 @@ def link_stream_using_brew_tag(brew_tag: str, stream_name: str, inherit: bool =
         product_variant, _ = ProductVariant.objects.get_or_create(name=variant)
         product_stream_node = ProductNode.objects.get(object_id=product_stream.pk)
         product_variant.pnodes.get_or_create(parent=product_stream_node)
+        # TODO: We already have a reference to product_stream
```

Comment:
This is a good idea. We can probably move the product_stream.save* functions to after the variants loop, which should improve the performance of this function quite a bit. Note that this function is only called if passing the 'stream' option to loadproductdata. I left the code in here, because I think it's a robust way of linking brew tags to product variants, but it's very resource intensive as it involves looking up every build in a specified brew tag from Errata Tool.